modules/TodoAppPage/components/DisplayTask/DisplayTask.tsx

```diff
@@ -0,0 +1,159 @@
+import { useState } from "react";
+import { useDisclosure } from "@mantine/hooks";
+import { Flex, Card, Text, Group, Button } from "@mantine/core";
+import { IconTrash } from "@tabler/icons-react";
+import dayjs from "dayjs";
+import Swal from "sweetalert2";
+
+import EditTask from "../EditTask/EditTask";
+import {
+  deleteTaskFromLocalStorage,
+  getSortedTasks,
+  markTaskComplete,
+} from "../../../../shared/utils/localStorage";
+import { TTask } from "@/shared/typedefs/types";
+import { EPriority, ETaskStatus } from "@/shared/typedefs/enums";
+
+type DisplayTaskProps = {
```

Comment:
```suggestion
type TDisplayTaskProps = {
```
Let's prepend types with `T`

Additionally, please create a types file for each component under that component's folder and put the types in that file.
In this case, create DisplayTask.types.ts under DisplayTask folder.